projects/kit/components/slider/slider.component.ts

```diff
@@ -60,6 +68,29 @@ export class TuiSliderComponent {
         }
     }
 
+    // TODO(v5): use signal inputs
+    @Input({
+        alias: 'segments',
+        transform: (segments: number[] | number) => {
+            if (Array.isArray(segments)) {
+                const arr = segments.map((segment) => segment * 100);
+
+                if (arr[0] !== 0) {
+                    arr.unshift(0);
```

Comment:
Do we actually need to mutate array ?
I think we could drop this logic (and process initial zero inside `getTicksGradient`).

Also, let's update documentation
```diff
- length in range [0; 1]
+ length in range (0; 1)
```

Passing `0` & `1` (as a position for the tick) seems senseless anyway